compose.prod.yaml

```
@@ -11,6 +11,7 @@ services:
       NODE_ENV: production
       GREEN_BITE_API_URL: http://green-bite:80
       DATABASE_URL: postgresql://postgres:<DATABASE PASSWORD>@db:5432/dont-spill-the-tea
+      GREEN_BITE_API_URL: http://green-bite:80
```

YAMLlint (1.38.0)

[error] duplication of key "GREEN_BITE_API_URL" in mapping

(key-duplicates)